drivers/mpu9250/MPU9250.cpp

```diff
@@ -442,8 +351,8 @@ void MPU9250::_measure()
 	}
 
 	// The FIFO buffer on the MPU is 512 bytes according to the datasheet, so let's use
-	// 36*14 = 504.
-	const unsigned buf_len = 36 * sizeof(fifo_packet);
+	// 36*size_of_fifo_packet.
+	const unsigned buf_len = 36 * size_of_fifo_packet;
```

Comment:
Won't this exceed 512 bytes when used with mag? I think 36 is only doable for the case without mag.